native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -157,6 +152,7 @@ SpineModel *SpineSkeletonInstance::updateRenderData() {
         _model->byteStride = sizeof(V3F_T2F_C4B);
     }
     collectMeshData();
+    globalMesh.textureID = "";
```

Comment:
Why reset the textureID to empty?

Reply:
globalMesh is a global variable. Next spine object may be affected by this variable.